gpkit/solution_array.py

```diff
@@ -302,7 +303,7 @@ def results_table(data, title, minval=0, printunits=True, fixedcols=True,
                                   model + r"}} \\"])
             oldmodel = model
         label = var.descr.get('label', '')
-        units = var.unitstr() if printunits else ""
+        units = var.unitstr(" [%s] ") if printunits else ""
```

Comment:
I thought we agreed to remove the square brackets awhile back? Why are they being added back in here?